addons/zookeeper/templates/cmpv.yaml

```diff
@@ -7,13 +7,18 @@ metadata:
 spec:
   compatibilityRules:
     - releases:
+        - 3.4.14
         - 3.6.4
         - 3.7.2
         - 3.8.4
         - 3.9.2
       compDefs:
         - zookeeper
   releases:
+    - name: 3.4.14
+      serviceVersion: 3.4.14
+      images:
+        zookeeper: {{ .Values.images.registry | default "docker.io" }}/{{ .Values.images.repository }}:3.4.14
```

Comment:
Synchronize with the main branch and keep the style consistent.

Reply:
just rebase main